desktop/src/test/java/bisq/desktop/GuiceSetupTest.java

```diff
@@ -131,6 +132,7 @@ public void testGuiceSetup() {
         assertSingleton(TradeEvents.class);
         assertSingleton(PriceAlert.class);
         assertSingleton(MarketAlerts.class);
+        assertSingleton(ChargeBackRisk.class);
```

Comment:
ChargebackRisk is stateless so it does not need to be a singleton. of course it does not hurt to be a singleton, but I think guice best practice for stateless services is to keep them as default scope.